python/ray/_private/services.py

```diff
@@ -1313,7 +1316,8 @@ def start_dashboard(require_dashboard,
             f"--port={port}", f"--port-retries={port_retries}",
             f"--redis-address={redis_address}", f"--temp-dir={temp_dir}",
             f"--log-dir={logdir}", f"--logging-rotate-bytes={max_bytes}",
-            f"--logging-rotate-backup-count={backup_count}"
+            f"--logging-rotate-backup-count={backup_count}",
+            f"--gcs-address={gcs_address}"
```

Comment:
I have been seening flags like `--gcs-address=None` in logs for start commands. It should be fine for now if we are not reading these flags. Eventually we would need to either delete the redis flag or avoid setting it when it is `None`.

Reply:
I agreed with you! But I think it's ok for now since we're going to remove them in near future and right now py args accept this. Let's try to make it work first and polish it later.